app/models/generic_file.rb

```diff
@@ -3,17 +3,17 @@ class GenericFile < ActiveFedora::Base
   include Sufia::GenericFile
 
   property :spatial, predicate: ::RDF::DC.spatial do |index|
-    index.as :stored_searchable, :facetable
+    index.as :stored_searchable, :symbol
```

Comment:
Yeah, you are right, I must have added it to see how the index would respond, but I forgot to change it back